packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -41,6 +41,7 @@ class ExpansionTile extends StatefulWidget {
     this.children = const <Widget>[],
     this.trailing,
     this.initiallyExpanded = false,
+    this.maintainState = false,
   }) : assert(initiallyExpanded != null),
        super(key: key);
```

Comment:
We'll probably want to assert that the default value has not been overridden with null.
```suggestion
       assert(maintainState != null),
       super(key: key);
```